### PR TITLE
Fix storage remove workspace message

### PR DESCRIPTION
**Pull request by rcvalerio**
closes https://github.com/inductiva/tasks/issues/502

**Web-api PR must be merged first**: https://github.com/inductiva/inductiva-web-api/pull/1349

### CODE REVIEW

File: inductiva/storage/storage.py [hunk 1]
```diff
@@ -256,27 +256,23 @@ def _list_files(root_path: str) -> Tuple[List[str], int]:
...
 
+    # Since we don't allow root files in workspaces it must be a directory
+    # otherwise path validation in the backend will give error
+    if '/' not in remote_dir:
+        remote_dir = remote_dir + '/'
     try:
-        api.delete_file(query_params={"path": remote_path},)
+        api.delete_file(query_params={"path": remote_dir},)
         logging.info("Workspace file(s) removed successfully.")
     except exceptions.ApiException as e:
```
Comment (Pbarbosa92) [inline inductiva/storage/storage.py:277]: Why do we just except just to raise it ?

Comment (rcvalerio) [reply inductiva/storage/storage.py]: Good point, I removed on other functions as well, can you confirm this is okay?

**Review by Pbarbosa92 — APPROVED** [review]
LGTM

**Comment by Pbarbosa92** [comment]
QA
LGTM
`inductiva.storage.remove_workspace("teste_upload/teste.jpg")`

```
import inductiva

input_dir = "/Users/paulobarbosa/Downloads/teste_upload"

# #missing overwride 
# inductiva.storage.upload("/Users/paulobarbosa/Downloads/teste_upload","teste_upload")


# # inductiva.storage.upload_from_url("https://storage.googleapis.com/media-newsinitiative/images/GO801_GNI_VerifyingPhotos_TitleCard.original.jpg","teste_upload",file_name="teste.jpg")

inductiva.storage.remove_workspace("teste_upload/teste.jpg")

# inductiva.storage.remove_workspace("teste_upload","olamundo/teste.jpg")

# import inductiva

# # Instantiate machine group
# # machine_group = inductiva.resources.MachineGroup("c2-standard-8", num_machines=2)
# # machine_group.start()

# machine_group = inductiva.resources.MPICluster("c2-standard-8", num_machines=2)
# machine_group.start()

# input_dir = inductiva.utils.download_from_url(
#     "https://storage.googleapis.com/inductiva-api-demo-files/fds-input-example.zip", unzip=True)

# custom_simulator = inductiva.simulators.CustomImage(container_image="docker://inductiva/kutu:fds_v6.8")

# task = custom_simulator.run(input_dir=None,
#                             commands=["ls"]
#                             ,remote_assets=["teste_upload"],
#                             on=machine_group)

# task.wait()
# task.download_outputs()

# machine_group.terminate()

```

**Pull request merged**